src/tasks/tasks.py

```diff
@@ -457,6 +462,10 @@ def update_metrics(self, logits, labels, tagmask=None):
 
 @register_task('cola-analysis', rel_path='CoLA/')
 class CoLAAnalysisTask(SingleClassificationTask):
```

Comment:
actually, could you put this into the `minimal_pairs` task file and rename it to something like `tasks/acceptability.py`? I'm trying to avoid growing `tasks.py` any more.

Reply:
Ok